assets/js/components/Accordion.js

```diff
@@ -25,10 +25,19 @@ import classnames from 'classnames';
 /**
  * WordPress dependencies
  */
-import { useState, useCallback } from '@wordpress/element';
+import { useState, useCallback, useEffect } from '@wordpress/element';
 
-export default function Accordion( { title, children, initialOpen } ) {
+export default function Accordion( { title, children, initialOpen, onOpen, onClose } ) {
 	const [ isActive, setActive ] = useState( !! initialOpen );
+
+	useEffect( () => {
+		if ( isActive && onOpen && typeof onOpen === 'function' ) {
+			onOpen();
+		} else if ( ! isActive && onClose && typeof onClose === 'function' ) {
+			onClose();
+		}
+	}, [ isActive ] );
```

Comment:
`onOpen` and `onClose` are properties, thus we need to add it to the dependencies list for this effect hook.

Reply:
Updated @eugene-manuilov